virus/p_intestivirus

```diff
@@ -0,0 +1,84 @@
+
+The proposal for a bacterial-mediated entry into the host organism is reflected by the results from another dataset, SRR5830110 (accessible via the SRA database). The sample containing the Posasimilis Intestivirus hit derives from the fecal community of Tule Elk. Blastx analysis matches the virus with Caudovirales sp. and Candidatus Scalindua sp. as the top hits. The presence of Caudovirales may strengthen the notion that our virus of interest operates in a bacteriophage-like manner. Moreover, Candidatus Scalindua is a genus of marine-associated bacteria first discovered in a waster water treatment plant [Schmid, 2003]. This highlights the capacity for Posasimilis intestinum to infect bacteria, specifically bacteria that may be found in the water supply of wild-life. 
+
+Related information pertaining to dataset runs can be found in the "MGY441 A3 Virus Seq Runs" csv file.
+
+[MGY441 A3 Virus Seq Runs.csv](virus/img/p_intestivirus/MGY441 A3 Virus Seq Runs.csv)
+Table 1: A 'Top Hits' Analysis for Each P.Intestivirus Accession Code Hit. 
+
+# Viral Genome 
+
+[MGY441 A3 Viral Genome.png](virus/img/p_intestivirus/MGY441 A3 Viral Genome.png)
+Figure 1. A) Proposed Genomic Architecture of the Posasimilaris Intestivirus. Conserved sequences were identified with Blastp and aligned with the coding amino acid sequence of the virus. Distance representation between conserved regions are not to scale. The presence of the 5’NTR and 3’NTR is based on the conserved 5’NTR and 3’NTR domains found in Picornavirales [Jiang, 2014]. Size and length is not to scale.  B) Predicted RNA Secondary Structure of Posasimilis Intestiviridae. The prediction was performed with the RNAfold tool, utilizing minimal free energy predictions only. 
```

Comment:
For all the Figure Legends, add a `>` in front of the line to make it "Quote Text". This will look nicer.